pkg/config/config.go

```diff
@@ -522,6 +524,45 @@ type Project struct {
 	GiteaURLs GiteaURLs `yaml:"gitea_urls,omitempty"`
 }
 
+func (p *Project) executeEnvTemplates() error {
```

Comment:
we have our own template package (`tmpl`) with some defaults et al, please use it instead.

example usages can be found in the codebase: https://github.com/goreleaser/goreleaser/search?q=%22tmpl.New%28%22

Reply:
also, the template should be evaluated in the build pipe, not here.

Reply:
Sure will make these changes,
thanks